jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/asserts/FileInputAssert.java

```diff
@@ -149,9 +143,9 @@ public FileInputAssert notDetailsHidden() {
 
     @JDIAction("Assert that '{name}' has loader height {0}")
     public FileInputAssert loaderHeightPx(int height) {
-        String actualLoaderHeight = element().getLoaderHeight();
-        jdiAssert(actualLoaderHeight, Matchers.equalTo(height + "px"), String.format("Actual element's loader height '%s'" +
-                " is not equal to expected %s", actualLoaderHeight, height + "px"));
+        int actualLoaderHeight = element().getLoaderHeight();
+        jdiAssert(actualLoaderHeight, Matchers.equalTo(height), String.format("Actual element's loader height '%s'" +
+                " is not equal to expected %s", actualLoaderHeight + "px", height + "px"));
```

Comment:
`
String.format("Actual element's loader height '%s' is not equal to expected %s", actualLoaderHeight + "px", height + "px")
->
String.format("Actual element's loader height '%s px' is not equal to expected '%s px'", actualLoaderHeight, height)`